prime-router/docs/getting-started/postgres-database.md

```diff
@@ -1,8 +1,266 @@
 # Postgres Database
 
-- Data restore - already exists
-- DB Failover restore - already exists
-- DB Flyaway repair - already exists
-- How to debug database queries (already exists)
-- SimpleReport data dictionary (already exists)
-  - Include content for the environment RS is running i.e local, staging, and prod
\ No newline at end of file
+## Data Restore
```

Comment:
Nothing in this section is related to Postgres Database? This is just covering azure storage container management? Seems like this should be in a different doc OR this doc needs a new title, like "Azure Management" but then anything specific to Postgres that does not relate to how to manage it in azure should get moved to another doc?